nodes/test/guardrails/test_all.py

```
@@ -0,0 +1,938 @@
+# ============================================================================
+
+
+class TestIInstanceLifecycle:
+    """Verify IInstance behaviors via direct method testing.
+
+    These tests import the IInstance class and mock the engine and
+    pipeline dependencies to verify behavior without a running server.
+    """
+
+    @staticmethod
+    def _load_iinstance_class():
```

Ruff (0.15.7)

[warning] Missing return type annotation for staticmethod `_load_iinstance_class`

(ANN205)